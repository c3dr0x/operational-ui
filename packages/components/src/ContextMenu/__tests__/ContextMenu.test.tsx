import * as React from "react"
import { render } from "enzyme"
import { ContextMenu as ThemelessContextMenu, Icon } from "../../index"
import wrapDefaultTheme from "../../utils/wrap-default-theme"

const ContextMenu = wrapDefaultTheme(ThemelessContextMenu)

describe("ContextMenu Component", () => {
  it("Should render", () => {
    const renderedComponent = render(
      <ContextMenu items={[ "Item 1", "Item 2", "Item 3" ]}>
        <Icon name="Document" size={16} />
      </ContextMenu>,
    )
    expect(renderedComponent).toMatchSnapshot()
  })
})
