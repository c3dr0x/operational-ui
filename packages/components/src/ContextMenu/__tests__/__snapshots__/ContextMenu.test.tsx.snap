// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContextMenu Component Should render 1`] = `
<div
  class="css-ar73w8-Messages"
/>
`;
